variant_annotation/README.md

```diff
@@ -32,5 +35,8 @@ optional arguments:
   --tumor_id TUMOR_ID
   --snpsift_countOpName SNPSIFT_COUNTOPNAME
   --snpsift_prevalOpName SNPSIFT_PREVALOPNAME
+  --opOncoKbMafName OPONCOKBMAFNAME
```

Comment:
@karthigayini  What does OP stand for?

Reply:
output - to indicate that it denotes output MAF file generated from oncoKB